donkeycar/parts/keras.py

```diff
@@ -317,9 +333,9 @@ def default_categorical(input_shape=(120, 160, 3), roi_crop=(0, 0)):
 
 def default_n_linear(num_outputs, input_shape=(120, 160, 3), roi_crop=(0, 0)):
 
-    drop = 0.1
+    drop = 0.2
```

Comment:
From my perspective it is not clear what and why this value was changed and what is the impact. Maybe adding a single line comment would be sufficient?

Reply:
This just increases the dropout in the linear model.